lib/storage-delegations.js

```diff
@@ -0,0 +1,193 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const assert = require('assert-plus');
+const bedrock = require('bedrock');
+const database = require('bedrock-mongodb');
+const {promisify} = require('util');
+const {util: {BedrockError}} = bedrock;
+
+bedrock.events.on('bedrock-mongodb.ready', async () => {
+  await promisify(database.openCollections)(['zcap-delegation']);
+
+  // FIXME: what are indexes for this collection?
+
+  await promisify(database.createIndexes)([{
+    collection: 'zcap-delegation',
+    fields: {invocationTarget: 1, id: 1},
+    options: {unique: true, background: false}
+  }, {
+    collection: 'zcap-delegation',
```

Comment:
Is our naming convention more like `delegated-zcap` or `delegatedZcap` for these things?

Reply:
If 'these things' refers to the collection name, it is my opinion that they should be namespaced by the module per my suggestion here : https://github.com/digitalbazaar/bedrock-zcap-storage/issues/1